tools/internal_ci/helper_scripts/prepare_build_windows.bat

```diff
@@ -35,6 +35,14 @@ python -m pip install google-api-python-client
 powershell -File src\csharp\install_dotnet_sdk.ps1
 set PATH=%LOCALAPPDATA%\Microsoft\dotnet;%PATH%
 
+@rem Install Python 3.8.0
+powershell -command "(New-Object Net.WebClient).DownloadFile('https://www.python.org/ftp/python/3.8.0/python-3.8.0-amd64.exe', 'C:\tools\python-3.8.0-amd64.exe')"
+powershell -command "& 'C:\tools\python-3.8.0-amd64.exe' /quiet InstallAllUsers=1 PrependPath=1 Include_test=0 TargetDir='C:\Python38'"
```

Comment:
For security purposes, we'll want to [verify these downloads with a checksum](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-filehash?view=powershell-6).

Reply:
Done in the script. Let's see if Kokoro is happy this time.